hera_pspec/pspecdata.py

```diff
@@ -4218,28 +4234,45 @@ def get_pspec_run_argparser():
     a = argparse.ArgumentParser(description="argument parser for pspecdata.pspec_run()")
 
     def list_of_int_tuples(v):
-        v = [tuple([int(_x) for _x in x.split()]) for x in v.split(",")]
+        if '~' in v:
```

Comment:
do you mind just putting the examples listed in the argparser here as well, e.g. `'0~0,1~1' --> [(0, 0), (1, 1), ...]` etc. and same for the other two functions below. thanks

Reply:
Sounds good! i've done this.